### PR TITLE
F upgrade storybook migrate dp toggle

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpToggle stories to v7.
- adjust Trans-Key.

### CODE REVIEW

**Comment by ahmad-demos** [comment]
> 

because of https://github.com/demos-europe/demosplan-ui/pull/420
I just wanted to make those small changes here, and later I'll do another PR to just make changes to a `translations.js` file!

**Comment by spiess-demos** [comment]
> > 
> 
> because of #420 I just wanted to make those small changes here, and later I'll do another PR to just make changes to a `translations.js` file!

i hope the removed lines (mandatoryfields...) are not overseen later.

**Review by spiess-demos — APPROVED** [review]
lgtm! translation keys will be hopefully fixed within another PR.

File: src/components/shared/translations.js [hunk 1]
```diff
@@ -21,9 +21,6 @@ const de = {
   contextualHelp: 'Kontexthilfe',
   entrySelected: "Eintrag ausgewählt",
   entriesSelected: "Einträge ausgewählt",
-  errorMandatoryFieldsDefault: 'Bitte füllen Sie alle Pflichtfelder korrekt aus. Erst dann können Sie Ihre Angaben speichern.',
-  errorMandatoryFieldsIntro: 'Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: ',
-  errorMandatoryFieldsOutro: '. Erst dann können Sie Ihre Angaben speichern.',
```
[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/shared/translations.js:26]: https://github.com/demos-europe/demosplan-ui/pull/454#pullrequestreview-1586825138
https://github.com/demos-europe/demosplan-ui/pull/454#issuecomment-1686182938

[the remark this answers is not in the document]
Comment (ahmad-demos) [reply src/components/shared/translations.js:26]: I'll do another PR to just make changes to a translations.js file!

**Review by salisdemos — APPROVED** [review]

**Pull request merged**